### DISCUSSION TITLE
render.py

### DISCUSSION THREAD

**Discussion by TimSousa1** [NVlabs/instant-ngp#907] [Q&A] [answered]
for some reason render.py seems to be missing!

**Comment by Tom94** [comment, marked as answer]
Hi, there is no render.py in this repository (and there never was). If you'd like to render a video, you need to use run.py as described in the README [here](https://github.com/NVlabs/instant-ngp#python-bindings).

**Reply by TimSousa1** [reply]
Yep! You're totally right! Sorry for the confusion.. eventually figured it out but forgot to take this 'issue' down!
Sorry for the inconvenience and thanks for the swift reply!
